posts/app.py

```diff
@@ -53,5 +53,26 @@ def get_all_posts_for_event(event_id):
     """Get all posts matching the event with the specified ID."""
 
 
-if __name__ == "__main__":
+def connect_to_mongodb():  # pragma: no cover
+    """Connect to MongoDB instance using env vars."""
+
+    class DBNotConnectedError(EnvironmentError):
+        """Raised when not able to connect to the db."""
+
+    class Thrower(object):
+        """Used to raise an exception on failed db connect."""
+
+        def __getattribute__(self, _):
+            raise DBNotConnectedError(
+                "Not able to find MONGODB_URI environmental variable")
```

Comment:
Minor nit: this use of "environmental" is non-standard, though correct English usage.

I'd switch it to "environment variable" even though that's a noun-noun pairing.

Reply:
Thanks for the clarification, done https://github.com/knative-portability/large-events/pull/9/commits/30a2993b7ebb2ee852338f35ef65d5f97bdea760